scripts/taginfo.js

```diff
@@ -30,29 +23,35 @@ function addNetworkTags(project, sprites) {
     }
     description += ".";
 
-    let icon = definition.backgroundImage || definition.norefImage;
-    if (Array.isArray(icon)) {
-      icon = icon[0];
```

Comment:
This version of the `icon` code was fine, except that it needs to refer to `spriteBlank` or `norefImage` and set `icon` to a full URL:

```js
icon = `https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/${icon[0]}.svg`;
```